.nvmrc

```
@@ -0,0 +1 @@
+22
```

22

... 22 what.

Why is there 22? What purpose does 22 solve? 🤯

---

LOL. nodejs 22

---

OK. this is fine.

looked at the nvmrc [spec](https://github.com/nvm-sh/nvm/issues/3265)...kinda. What a weird file.

```suggestion
v22
```

Should work as well, but it doesn't matter, 22 works.